lib/routes/routeBackbeat.js

```diff
@@ -754,22 +785,54 @@ function deleteObjectTagging(request, response, log, callback) {
         dataStoreVersionId, log, callback);
 }
 
+function _batchDeleteIfUnmodifiedSince(
+    request, response, locations, log, callback) {
+        return async.waterfall([
+            next => {
+                const reqUids = log.getSerializedUids();
+                return dataClient.head(locations, reqUids, next);
+            },
+            (data, next) => {
+                const { headers } = request;
+                if (headers['if-unmodified-since'] !== data.LastModified) {
+                    return next(null);
```

Comment:
`return next()`

It could be appropriate to add an info log with object info and both LastModified and if-unmodified-since values, so we know that this condition occurred on this object, since we silently skip but it's not a regular operation.